source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -58,6 +58,21 @@ results:
 This method is often used when your query filter will match one
 document, such as when filtering by a unique index.
 
+Available
+~~~~~~~~~
+
+Use the ``available()`` method to retrieve the amount of results
+available locally without blocking:
```

Comment:
Nitpick: it could be more clear if the explanation of the method "available" didn't include the word "available". Instead, consider using different words to describe it.